<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmDialog</class>
 <widget class="QDialog" name="ConfirmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>확인하시겠습니까?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
